src/tests/unit/tests/electron/views/screenshot/screenshot-view.test.tsx

```diff
@@ -1,19 +1,43 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
 import { ScreenshotView } from 'electron/views/screenshot/screenshot-view';
 import { ScreenshotViewModel } from 'electron/views/screenshot/screenshot-view-model';
 import { shallow } from 'enzyme';
 import * as React from 'react';
 
 describe('ScreenshotView', () => {
-    it('renders', () => {
-        const viewModel: ScreenshotViewModel = {
-            screenshotData: null,
-            highlightBoxRectangles: [],
-            deviceName: null,
-        };
-        const wrapper = shallow(<ScreenshotView viewModel={viewModel} />);
+    describe('renders', () => {
+        it('when passed a null value for screenshotData', () => {
```

Comment:
The view model provider already has the responsibility of normalizing "empty" screenshot data as null, so I'm not sure we need the other 2 cases here; if we do want them, it would probably be a little easier to read to use `.each`, like:

```ts
const emptyScreenshotDataCases: ScreenshotData[] = [null, undefined, {} as ScreenshotData];
it.each(emptyScreenshotDataCases)('when passed empty screenshotData %p', (screenshotData: ScreenshotData) => {
    // the same test implementation, but using the passed in screenshotData
});
```